RecoMET/METAlgorithms/src/HcalNoiseAlgo.cc

```diff
@@ -89,7 +89,7 @@ CommonHcalNoiseRBXData::CommonHcalNoiseRBXData(const reco::HcalNoiseRBX& rbx, do
     double eme=it1->caloTowerEmE();
     double hade=it1->recHitEnergy(minRecHitE);
     double emf=(eme+hade)==0 ? 999 : eme/(eme+hade);
-    if(HPDEMF_ > emf) emf = HPDEMF_;
+    if(HPDEMF_ > emf) HPDEMF_ = emf;
```

Comment:
hi @bsunanda - please confirm this logic change is correct (that you want to set HPDEMF_ to the minimum value of emf